pyproject.toml

```diff
@@ -0,0 +1,132 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "nettacker"
+maintainers = [ { name = "OWASP" } ]
+version = "0.3.1"
+description = "Automated Penetration Testing Framework - Open-Source Vulnerability Scanner - Vulnerability Management"
+keywords = [
+    "python", 
+    "security", 
+    "automation",
+    "scanner",
+    "bruteforce",
+    "owasp",
+    "penetration-testing",
+    "pentesting",
+    "cve",
+    "network-analysis",
+    "vulnerability-management",
+    "vulnerability-scanners",
+    "information-gathering",
+    "portscanner ",
+    "security-tools",
+    "vulnerability-scanner",
+    "penetration-testing-framework",
+    "hacking-tools", 
+    "pentesting-tools",
+    "cves",
+]
+readme = "README.md"
+requires-python = ">=3.7"
+license = { file = "LICENSE" }
+dependencies = [
+    "argparse==1.4.0",
+    "netaddr==0.9.0",
+    "ipaddr==2.2.0",
+    "requests==2.31.0",
+    "aiohttp==3.8.5",
+    "asyncio==3.4.3",
+    "paramiko==3.3.1",
+    "texttable==1.6.7",
+    "PySocks==1.7.1" ,
+    "pyOpenSSL==23.2.0",
+    "flask==3.0.0",
+    "SQLAlchemy>=1.4.43",
+    "py3DNS==4.0.0" ,
+    "numpy==1.26.0",
+    "terminable_thread==0.7.1",
+    "PyYAML==6.0.1",
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache-2.0 license",
+    "Operating System :: OS Independent",
+    "Development Status :: 5 - Production/Stable",
```

Comment:
It deserves the security topic:
```suggestion
    "Development Status :: 5 - Production/Stable",
    "Topic :: Security",
```

You can find more here -- https://pypi.org/pypi?%3Aaction=list_classifiers